boards/arm/rcar_m3ulcb/rcar_m3ulcb_cr7_defconfig

```diff
@@ -0,0 +1,20 @@
+CONFIG_SOC_R8A77961=y
+CONFIG_SOC_SERIES_RCAR_GEN3=y
+CONFIG_BOARD_RCAR_M3ULCB_CR7=y
+CONFIG_CLOCK_CONTROL=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=32000
+CONFIG_RCAR_CMT_TIMER=y
+CONFIG_CONSOLE=y
+CONFIG_RAM_CONSOLE=y
+CONFIG_FLASH_SIZE=0
+CONFIG_FLASH_BASE_ADDRESS=0
+CONFIG_GPIO=y
+CONFIG_GPIO_RCAR=y
+CONFIG_SERIAL=y
+CONFIG_UART_RCAR=y
+CONFIG_UART_CONSOLE=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_CAN=y
+CONFIG_CAN_RCAR=y
+CONFIG_I2C=y
+CONFIG_I2C_RCAR=y
```

Comment:
You should not enable these in the defconfig. Only mandatory, essential features should be enabled here.

Reply:
Another comment: You should only need
```
CONFIG_CAN=y
CONFIG_I2C=y
```
Otherwise, this means the RCAR Kconfig has some issues. You can compare with other drivers to find a solution.

Reply:
@henrikbrixandersen @gmarull something like that ?
https://github.com/zephyrproject-rtos/zephyr/pull/42027